src/components/PokemonCart.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react'
+
+export default function PokemonCart({ cart, removePokemonFromCart }) {
+  return (
+    <>
+      <h4>Pokemon Cart</h4>
+      <p>Total: ${cart.reduce((total, pokemon) => total + pokemon.price, 0)}</p>
```

Comment:
Because this operation has more complexity, you can add this value to a variable and just interpolate the value:

```
 const total = cart.reduce((total, pokemon) => total + pokemon.price, 0)
 ...
<p>{`Total: ${total}`}</p>
```

Reply:
Yeah that's looks better, thanks